flake8_import_order/styles.py

```diff
@@ -171,13 +171,13 @@ def sorted_names(names):
 
     @staticmethod
     def name_key(name):
-        return (name.lower(), name)
+        return name.lower(), name
```

Comment:
These are all personal stylistic changes, I'm not sure they belong.

Reply:
I agree they are mostly stylistic changes, thus the PR title minor cleanups.

The one change of substance is making parse options more DRY.  
I made this change intentionally because I have some very significant changes I am working on and it would simplify additions of new options.

Essentially without getting too much into it here I am changing to a configuration based system where the different styles are defined as a set of standard options instead of subclasses.

It would allow users to customize the import-order without necessarily requiring a new style to be created.  For example the PyCharm style recently implemented only covers a single way PyCharm can be configured and could have been implemented as `smarkets` with case-sensitivity option.

Feel free to close the PR if you feel they are not appropriate.